contrib/keyd/patches/no-systemd.patch

```diff
@@ -0,0 +1,15 @@
+--- a/Makefile
```

Comment:
this patch doesn't do anything? `$(DESTDIR)$(PREFIX)/lib/systemd/` doesn't exist so it doesn't actually install a service file and goes to else, so then it wouldn't have to be patched

Reply:
Installation was failing because my system does have a directory /usr/lib/systemd with 
../systemd/user/dconf.service and 
../systemd/user/org.gnome.Evince.service

Reply:
it also works fine just removing this, i just checked

Reply:
no that's not the error message. What I mean is that line 50 of the makefile
@if [ -e $(DESTDIR)$(PREFIX)/lib/systemd/ ]; then \
fails on my system because I have a directory /usr/lib/systemd/

This is the error message.
$ doas gmake install
install: /usr/lib/systemd/system/keyd.service: No such file or directory
gmake: *** [Makefile:50: install] Error 71

Reply:
that happens because it uses `install -D`, but that never happens in actual build so it doesn't matter

Reply:
removed and package builds successfully.